### DISCUSSION TITLE
Play Structure > Progression Structure > Party

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#74] [Framework Discussions]
# Play Structure > Progression Structure > Party

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
Each in-game match progresses equally for each member of the team

**Comment by Tomathy-dev** [comment]
Players often form groups to participate in both PvE (Player vs Environment) or PvP (Player vs Player). Both provide progression by rewarding Exp (Experience), in-game currency (gold, silver and copper) and valuable loot. These groups can vary between 2 to 30 players, depending on the content.
